packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -561,6 +561,71 @@ export class PipelineEditor extends React.Component<
     });
   }
 
+  async handleLoadPipeline(): Promise<void> {
+    this.widgetContext.ready.then(() => {
+      let pipelineJson = this.widgetContext.model.toJSON();
+      const pipelineVersion: number =
+        +Utils.getPipelineAppdataField(pipelineJson, 'version') || 0;
+      if (pipelineVersion !== PIPELINE_CURRENT_VERSION) {
+        // pipeline version and current version are divergent
+        if (pipelineVersion > PIPELINE_CURRENT_VERSION) {
+          // in this case, pipeline was last edited in a "more recent release" and
+          // the user should update his version of Elyra to consume the pipeline
+          showDialog({
+            title: 'Load pipeline failed!',
+            body: (
+              <p>
+                This pipeline corresponds to a more recent version of Elyra{' '}
+                <br />
+                and cannot be used until Elyra has been upgraded.
+              </p>
+            ),
+            buttons: [Dialog.okButton()]
+          });
+          if (this.app.shell.currentWidget) {
```

Comment:
We could make this a pipeline command `closePipelineEditor` to make it consistent with the other ones, then just call `this.app.commands.execute(commandIDs.closePipelineEditor);`.
Same for line 616